code/game/objects/items/kitchen.dm

```diff
@@ -190,20 +190,29 @@
 	icon_state = "knife_cyborg"
 	desc = "A cyborg-mounted plasteel knife. Extremely sharp and durable."
 
-/obj/item/kitchen/knife/carrotshiv
-	name = "carrot shiv"
-	icon_state = "carrotshiv"
-	item_state = "carrotshiv"

```

Comment:
You need to replace all the old path with the new path on snowcabin.dmm. Its failing travis.

Reply:
Thanks for the catch